tests/topologyDescription/topologyDescription-var_export-001.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+MongoDB\Driver\TopologyDescription: var_export()
```

Comment:
Since `__set_state` is not implemented on this class (for the same reason it's not serializable), I'd suggest we avoid testing `var_export` and stick to `var_dump` and the "debug" test(s). This file can be deleted.